portfolio/src/main/webapp/js/script.js

```diff
@@ -65,6 +83,9 @@ function initMap() {
   });
 }
 
+// Initialize map markers.
+putShawarmaMarkers();
+
```

Comment:
There is a chance the map is not initialized yet, since you do it from a callback.

You can use [setTimeout](https://www.w3schools.com/jsref/met_win_settimeout.asp) to check if the map was initialized and delay the addition of the shawarma places until it does.